common/tools/dev-tool/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "ES2021",
     "module": "commonjs",
     "moduleResolution": "node",
-    "noEmit": true,
+    "outDir": "./dist",
```

Comment:
why are we compiling it?

Reply:
Without output dev-tool will not be cached so it will always be built.  I am on the fence: either way is fine to me as it only takes a couple seconds to build.

Reply:
Let's have the output to save build time with the build cache